go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/minio/simdjson-go
 go 1.13
```

Comment:
If you are doing that @HeavyRain266 you need to make sure the minimum go is 1.16

Reply:
sorry, I forgot to look at this place.